src/software/abinit.jl

```diff
@@ -962,3 +962,29 @@ function write_abinit_modes(modes::Array{SVector{6,Float64}}, energies::Vector{F
         end
     end
 end
+
+"""
+    read_abinit_anaddb_PHDOS(filename::AbstractString) -> Tuple{DensityOfStates, Vector{ProjectedDensityOfStates}}
```

Comment:
By convention, I try to leave a blank line between the function name and its documentation below.

Also, in order to keep lines below 100 characters, I generally put the return type on a new line immediately after the function definition.